_posts/2025-02-26-PRISM.md

```diff
@@ -0,0 +1,21 @@
+---
+layout: post
+title: We published the PRISM framework
+author: reto-weber
+---
+We ([Bertrand Meyer][bm] and [Reto Weber][rw]) have published an extended and revised version of the [theory of programs][top], now called [PRISM][pm]. 
+This work involved a complete rework from the ground up, refining definitions and, 
+most importantly, rigorously proving the claims presented in the paper.
+
+We replaced previously disproven theorems with correct formulations and provided a precise definition of concurrency. 
+Additionally, during our research, we encountered the concept of "Choice Normal Form" (CNF), 
+which proved valuable for establishing various properties and describing numerous behaviors.
+
+I thoroughly enjoyed working on the proofs using Isabelle/HOL and will continue contributing to this project in the future. 
+For more information and updates on PRISM, please visit our [designated website][PRISM].
+
+[top]: https://arxiv.org/abs/1507.00723
+[bm]: /people/bertrand-meyer
+[rw]: /people/reto-weber
+[pm]: /publications/PRISM
+[PRISM]: /PRISM
```

Comment:
This is probably an external link (separate website). Maybe it is better to make it more explicit
```suggestion
[PRISM]: https://se.constructor.ch/PRISM
```